README.md

```diff
@@ -34,6 +34,12 @@ Yarn:
 yarn add sharetribe-flex-sdk
 ```
 
+## Getting client ID
+
+Client ID is required to create new SDK instance. 
+See [the doc](https://www.sharetribe.com/docs/background/applications/)
```

Comment:
```suggestion
See [this article](https://www.sharetribe.com/docs/background/applications/)
```

Reply:
Actually, like my comment on the Integration SDK, maybe this paragraph could just go in the Usage section below the code example?

Reply:
True, now it may stand out a bit in the README, it could be along the lines
```
// Create new SDK instance
// To obtain a client ID, see Applications in Flex Console
const sdk = sharetribeSdk.createInstance({
```
And maybe the docs site could have a more thorough explanation?

Reply:
Kinda accepted suggestions from the both.

> And maybe the docs site could have a more thorough explanation?

I agree.

Reply:
DA team, Would you consider updating the article in the future? 
https://www.sharetribe.com/docs/background/applications/

@lyyder @SariSaar @shareoc

Reply:
@wontheone1 I mean the SDK docs site: https://sharetribe.github.io/flex-sdk-js/

Instructions for updating it are found here: https://sharetribe.github.io/flex-sdk-js/developing-sdk.html#update-the-github-pages

Reply:
@lyyder https://sharetribe.github.io/flex-sdk-js/ is already updated with the new README. Do you mean we should add more contents only on the Github page (resulting in difference in Github README contents and the Github Page)?

Reply:
@wontheone1 I was thinking more like featuring it in some of the Getting Started section pages in SDK documentation. Then again, I don't think we've received too many messages on people using the SDK and not getting a hold of the client ID, so I think the current code snippet change in the README will do just fine.

Reply:
Aha, yeah I agree this might be enough. Thanks!

Reply:
But IMO https://www.sharetribe.com/docs/background/applications/ still has not enough contents for the record. Probably we will come back to that in the future.